cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +96,38 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftOrg,
+		"draft-org",
+		"",
+		"",
+		"a Github organization or user where the Release Notes PR will be created",
+	)
+
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftRepo,
+		"draft-repo",
+		"",
+		"",
+		"the name of a Github repository where the Release Notes PR will be created",
+	)
+
+	releaseNotesCmd.PersistentFlags().BoolVarP(
+		&releaseNotesOpts.createDraftPR,
+		"create-draft-pr",
+		"",
+		false,
+		"create the Release Notes draft PR. --draft-org and --draft-repo must be set along with this option",
+	)
+
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.outputDir,
+		"output-dir",
+		"o",
+		"",
```

Comment:
```suggestion
		".",
```

Reply:
Done the default is now . and I've ported the --format option for your previous version. If you run it with --create-draft-pr it will not render the release notes in the current directory